ui/app/utils/classes/poll-logger.js

```diff
@@ -30,11 +30,15 @@ export default EmberObject.extend(AbstractLogger, {
 
       if (text) {
         const lines = text.replace(/\}\{/g, '}\n{').split('\n');
-        const frames = lines.map(line => JSON.parse(line));
-        frames.forEach(frame => (frame.Data = window.atob(frame.Data)));
-
-        this.set('endOffset', frames[frames.length - 1].Offset);
-        this.get('write')(frames.mapBy('Data').join(''));
+        const frames = lines
+          .map(line => JSON.parse(line))
+          .filter(frame => frame.Data != null && frame.Offset != null);
+
+        if (frames.length) {
+          frames.forEach(frame => (frame.Data = window.atob(frame.Data)));
```

Comment:
What is it you're using for `atob` here?

Reply:
[The browser](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob). No dependency needed!